### PR TITLE
Decentraland contracts testnet addresses

### PR DESCRIPTION
**Pull request by marcelomorgado**
### Issue link

Refs: #135

### Types of changes

Technical debt (a code change that doesn't fix a bug or add a feature but makes something clearer for devs)

### Notes

I'm not sure if this is enough to close the issue.

### CODE REVIEW

File: packages/tasit-contracts/decentraland/scripts/install.sh [hunk 1]
```diff
@@ -9,26 +9,32 @@ fi
...
     then
         rm -rf $REPO_DIR;
         git clone https://github.com/decentraland/$REPO.git $REPO_DIR;
-        cd $REPO_DIR && git fetch origin $COMMIT && git reset --hard FETCH_HEAD
+        cd $REPO_DIR && git fetch origin $COMMIT && git reset --hard FETCH_HEAD;
+    fi
 
+    if [ $IS_NPM ];
+    then
+        npm i --prefix $REPO_DIR;
```
Comment (pcowgill) [inline packages/tasit-contracts/decentraland/scripts/install.sh]: What's the idea behind this `is_npm` in plain English? I think I see what's happening here, but I wanted to make sure.

File: packages/tasit-sdk/src/testHelpers/helpers.js [hunk 2]
```diff
@@ -8,11 +8,15 @@ import { abi as landProxyABI } from "./abi/LANDProxy.json";
...
 import { abi as markplaceABI } from "./abi/Marketplace.json";
 
-const manaAddress = "0xb32939da0c44bf255b9810421a55be095f9bb3f4";
-const landAddress = "0x6191bc768c2339da9eab9e589fc8bf0b3ab80975";
-const landProxyAddress = "0x773f11ed472aa43e4ebaa963bcfbbea5a10c1bbd";
-const estateAddress = "0x41b598a2c618b59b74540ac3afffb32f7971b37a";
-const marketplaceAddress = "0x07c0e972064e5c05f7b3596d81de1afd35457eae";
+import { local as localAddresses } from "../../../tasit-contracts/decentraland/addresses";
+const {
+  MANAToken: manaAddress,
```
Comment (pcowgill) [inline packages/tasit-sdk/src/testHelpers/helpers.js]: I wonder if `MANA_ADDRESS` would make more sense here

Comment (pcowgill) [inline packages/tasit-sdk/src/testHelpers/helpers.js]:
```suggestion
  MANAToken: MANA_ADDRESS,
```

Comment (pcowgill) [reply packages/tasit-sdk/src/testHelpers/helpers.js]: I'm resolving this since the suggestion in another thread already covers this idea

File: packages/tasit-contracts/decentraland/scripts/install.sh (hunk 1 above)
Comment (marcelomorgado) [reply packages/tasit-contracts/decentraland/scripts/install.sh]: If the repo is a npm project, run `npm install`. The `contracts` repo isn't a npm project, because of that I've added this condition.

Comment (pcowgill) [reply packages/tasit-contracts/decentraland/scripts/install.sh]: Oh all the other ones are? I could have sworn I couldn’t find them the other day. But I guess I could have checked their package.json first

When you say contracts repo do you mean `marketplace-contracts`

Comment (marcelomorgado) [reply packages/tasit-contracts/decentraland/scripts/install.sh]: No, not really.
/contract is the Decentraland Contract Addresses JSON (https://github.com/decentraland/contracts) it's only a data repo.

Comment (marcelomorgado) [reply packages/tasit-contracts/decentraland/scripts/install.sh]: Yes, all others repos are npm / truffle projects.

File: packages/tasit-sdk/src/testHelpers/helpers.js (hunk 2 above)
Comment (marcelomorgado) [reply packages/tasit-sdk/src/testHelpers/helpers.js]: Done.

File: packages/tasit-contracts/decentraland/scripts/install.sh (hunk 1 above)
Comment (pcowgill) [reply packages/tasit-contracts/decentraland/scripts/install.sh]: Ah ok, thanks for the explanation. Cool! I don’t think we can count on that being the case for other projects, but we might as well take advantage of it for this one!

File: packages/tasit-sdk/src/testHelpers/helpers.js (hunk 2 above)
Comment (pcowgill) [reply packages/tasit-sdk/src/testHelpers/helpers.js]: Thanks. Resolving.
